dockerfiles/dataset/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:14.04
```

Comment:
When I try to use ```python:2.7.12-slim``` base image, ```lsb_release --release --short``` prints ```8.6```, I guess it will not find flocker in the aws repo. So I am using ubuntu:16.04 for now.